### DISCUSSION TITLE
Css not being applied on github pages

### DISCUSSION THREAD

**Discussion by sampad-singha** [community/community#21641] [Other Feature Feedback, Questions, & Ideas]
<p>Hi.<br>
I created a simple website using Html and CSS. But, when I hosted the website, only HTML is being applied.<br>
code: <a href="https://github.com/sampad201199/Personal_Website" class="inline-onebox" rel="noopener nofollow ugc">GitHub - sampad201199/Personal_Website</a><br>
website: <a href="https://sampad201199.github.io/Personal_Website/" class="inline-onebox" rel="noopener nofollow ugc">Sampad Singha</a></p>

**Comment by Medowar** [comment, marked as answer]
<p>you capitalized the Foldername CSS, either link to the CSS with uppercase or rename the folder to lowercase.</p>
<p>URLs are Case-Sensitive</p>

**Comment by sampad-singha** [comment]
<p>Thank you, <a class="mention" href="/medowar">@Medowar</a>. It worked.</p>
